mx.sulong/mx_sulong.py

```diff
@@ -1221,4 +1236,5 @@ def sulongBuild(args=None):
     'su-get-llvm-program' : [getLLVMProgramPath, ''],
     'su-get-gcc-program' : [getGCCProgramPath, ''],
     'su-compile-sulong-suite' : [compileSulongSuite, ''],
+    'su-compile-llvm-suite' : [compileLLVMSuite, ''],
```

Comment:
Would it make sense if we had a `su-compile` command that takes the arguments `sulong` and `llvm`? Otherwise we just keep expanding the mx command with each newly supported test suite.